doc/en/customize.rst

```diff
@@ -97,6 +97,19 @@ check for ini-files as follows::
 .. _`how to change command line options defaults`:
 .. _`adding default options`:
 
+Change which files are added to sys.path
+-----------------------------------------------
+
+If you place a conftest.py file in the root directory of your project
+(as determined by pytest, see above.) pytest will run tests against 
+the code below that directory by adding it to your sys.path instead of 
```

Comment:
Just now getting back to this...  Should I remove these paragraphs from the pull request then?  They would have saved me a heck of a lot of grief.  Perhaps the conftest.py thing should be mentioned as an unsupported side effect somehow, instead of a supported feature?  

Either way, the thing I really think should be more clearly stated is that the user is expected to install the python code to be tested in develop mode.  As a newb coming from using nose's sys.path hackery, that pattern was not at all obvious to me.  I'm happy to shelve the conftest.py piece of this and focus on documenting the develop mode pattern.